x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/use_find_attack_discoveries/index.ts

```diff
@@ -50,10 +51,11 @@ interface PageParam {
   perPage?: number;
 }
 
-const DEFAULT_PAGE = 1; // CAUTION: sever-side API uses a 1-based page index convention (for consistency with similar existing APIs)
+const DEFAULT_PAGE = 1; // CAUTION: server-side API uses a 1-based page index convention (for consistency with similar existing APIs)
```

Comment:
This PR is not tagged as being backported to `8.19`, but consider backporting this change to simplify future backports and testing, (avoiding diverging implementations).